torch/testing/_internal/common_optimizers.py

```diff
@@ -1162,6 +1163,23 @@ def _get_optim_inputs_including_global_cliquey_kwargs(
                 lambda opt: ReduceLROnPlateau(opt, threshold=1e-4),
             ],
         ),
+        decorators=(
+            DecorateInfo(
+                #  Note on tolerances:
+                #  difference comes from the fact that the non fused kernel have
+                #  more dtype cast operations. We have another test test_fused_cpu_matches_cuda
+                #  to make sure there is no discrepancies between cuda fused kernel
+                #  and cpu fused kernel
```

Comment:
well..we don't have another test here so this part of the comment should be removed

Reply:
We may still need this. `test_fused_matches_forloop` have larger numerical gap. So we have another test `test_fused_cpu_matches_cuda` to compare results between cuda fused kernel and cpu fused kernel.

Reply:
Right, but this test doesn't exist for adagrad since there is no cuda impl for it right now, so the comment isn't true

Reply:
Thanks for advising, I did not realize it, thanks!